src/App.tsx

```diff
@@ -76,18 +66,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && loading && !error && posts.length === 0 && (
```

Comment:
```suggestion
                {author && loading && !error && !posts.length && (
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**